rasterview.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="image/pwg-raster">
+    <comment>PWG raster image</comment>
+    <glob pattern="*.pwg"/>
+  </mime-type>
+  <mime-type type="image/urf">
+    <comment>Apple raster image</comment>
+    <glob pattern="*.apple"/>
```

Comment:
I think we can also allow *.urf at this point (*.apple was an attempt to appease Apple management back in the day...)